content/docs/command-reference/commit.md

```diff
@@ -140,83 +140,83 @@ stage with `dvc run --no-commit`, or reproduce an entire pipeline using
 development of the stage is finished, `dvc commit` can be used to store data
 files in the cache.
 
-In the `featurize.dvc` stage, `src/featurize.py` is executed. A useful change to
-make is adjusting a parameter to `CountVectorizer` in that script. Namely,
-adjusting the `max_features` value in the line below changes the resulting
-model:
+In the `featurize` stage, `src/featurization.py` is executed. A useful change to
+make is adjusting the parameters for that script. The parameters are defined in
+the `params.yaml` file. Updating the value of the `max_features` param to
+6000 changes the resulting model:
 
-```python
-bag_of_words = CountVectorizer(stop_words='english',
-            max_features=6000, ngram_range=(1, 2))
+```yaml
+featurize:
+  max_features: 6000
+  ngrams: 2
 ```
 
-This edit introduces a change that would cause the `featurize.dvc`, `train.dvc`
-and `evaluate.dvc` stages to execute if we ran `dvc repro`. But if we want to
-try several values for `max_features` and save only the best result to the
-cache, we can run it like this:
+This edit introduces a change that would cause the `featurize`, `train` and
+`evaluate` stages to execute if we ran `dvc repro`. But if we want to try
+several values for `max_features` and save only the best result to the cache, we
+can run it like this:
 
 ```dvc
-$ dvc repro --no-commit evaluate.dvc
+$ dvc repro --no-commit evaluate
 ```
```

Comment:
I think we don't need `evaluate` anywhere here. Please test, re-read, etc.

Reply:
As mentioned in https://github.com/iterative/dvc.org/pull/1626#pullrequestreview-460959819

Reply:
Yes this can be removed. `repro` executes all three stages except first one in both cases(with and without evaluate)
```
$dvc repro --no-commit

'data/data.xml.dvc' didn't change, skipping
Stage 'prepare' didn't change, skipping
Running stage 'featurize' with command:
	python src/featurization.py data/prepared data/features
...
Updating lock file 'dvc.lock'

Running stage 'train' with command:
	python src/train.py data/features model.pkl
...
Updating lock file 'dvc.lock'

Running stage 'evaluate' with command:
	python src/evaluate.py model.pkl data/features scores.json prc.json
Updating lock file 'dvc.lock'
```

Reply:
So `evaluate` IS executed? Then the text above was correct, just the command doesn't need to specify a target anymore (that's a 1.x update).

Reply:
Yeah I remember when I first started working on this PR around that time(3 weeks before), `evaluate` will only execute if it was mentioned in target. Using only `repro --no-commit` skipped executing `evaluate`. After 1.x update this is fixed.